### DISCUSSION TITLE
Force update a Client object

### DISCUSSION THREAD

**Discussion by ethanwells** [mmohades/Venmo#70] [Q&A]
I have these two lines of code. I am continuously looping to scrape any new transactions by the user with the user_id. How do I force update cache?

```
client = Client(access_token=access_token)
transactions = client.user.get_user_transactions(user_id='xxxxxxxxxx')
```

**Comment by mmohades** [comment]
`get_user_transactions()` doesn't cache anything, it makes a new request to the API on every function call.
If you're trying to reach older transactions, try `transactions.get_next_page()` as shown [here](https://github.com/mmohades/Venmo#pagination).
